scalableminds/webknossos
# download hybrid tracings as nml/zip

### URL of deployed dev instance (used for testing):
- https://hybriddownload.webknossos.xyz

### Steps to test:
- in application.conf, set hybridTracings = true in features block
- create hybrid tracing, create some nodes, brush some voxels
- download, you should get a zipfile with one nml (with nodes *and* volume reference *and not* duplicate parameters/metadata)
- nmls for volume-only oder skeleton-only should still work and still contain parameters/metadata

### Issues:
- first half of #2987 

#### TODO:
- [x] Adapt Unit Tests

------
